tests/plugins/pastefromword/manual/keepemptymargins.md

```diff
@@ -0,0 +1,20 @@
+@bender-tags: feature, 4.11.4, 2935, pastefromword
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, undo, pastefromword, sourcearea
+
+1. Copy and paste content of following file into both editors:
+[Vertical_margin.docx](../generated/_fixtures/Vertical_margin/Vertical_margin.docx)
+1. Open source mode.
+
+## Expected
+
+- The first editor: the first `p` element has styles:
+```margin-right:0; margin-bottom:0; margin-left:0;```
```

Comment:
The styles in IE11 are a little different which might be misleading
<img width="1146" alt="Screen Shot 2019-04-11 at 18 05 07" src="https://user-images.githubusercontent.com/1061942/55972857-9a8ed080-5c84-11e9-8dca-d2dd0664094e.png">